packages/contracts/src/interfaces/IStakeManager.sol

```diff
@@ -73,6 +73,30 @@ interface IStakeManager {
         address indexed burnAddress
     );
 
+    /// @notice Emitted when a `burnAddress` is changed.
+    event DevAddressSet(
+        address indexed burnAddress
+    );
+
+    /// @notice
+    event RelayAbandoned(
```

Comment:
"RelayServer" or "Relayer".

we use "Relay" (a verb) to refer to a "server" (noun)